docs/ReleaseProcess.md

```diff
@@ -72,19 +120,21 @@ Servicing a release has two main scenarios:
     git mergetool
     ```
 1. Verify all the changes then commit all in the working branch.
-    >NOTE: The formatting rules may complain about the formatting of the generated `.c` files from the script above. In this case, override them with the following (so they'll work with the `bpf2c_tests` verifying their content):
+    >NOTE: The formatting rules may complain about the formatting of the generated `.c` files from the script above. In this case, override them with the
+    following (so they'll work with the `bpf2c_tests` verifying their content):
     >```bash
     >git commit --no-verify
     >```
-1. Create a **Draft** pull-request for your working branch into the main repo's "`release/X.Y`" branch, and title the PR as *"Release v`X.Y.Z`"* (replace "`X.Y.Z`" with the version number being released).
-1. Wait for  the CI/CD pipeline for the PR to complete successfully.
-1. Submit the PR for review (from its draft state), and wait for it to be approved and merged into the main repo's "`release/X.Y`" branch.
-1. Publish the release as per the "[Publishing a Release](ReleaseProcess.md#publishing-a-release)" process.
+1. Create a pull-request from the topic branch into the original repo's "`release/X.Y`" branch, and title the PR as *"Release v`X.Y.Z`"* where "`Z`" is the patch
+version.
+1. Once the PR is approved and merged into the "`release/X.Y`" branch in the original repo, create a tag for the latest commit as "`Release-vX.Y.Z`".
+1. Publish the patch release as per the [Publishing a Release](ReleaseProcess.md#publishing-a-release) process.
 
 
 ### Updating the main brach with patches/hot-fixes from a Release branch (*Reverse/Backwards Integration*)
 
->IMPORTANT! Normally, this should be done by the release manager **in VERY RARE scenarios** (and it's also likely an indication there's been a failure in the releasing process), but if you are a contributor and have been asked to do this, here are the steps to be followed:
+>IMPORTANT! Normally, this should be done by the release manager **in VERY RARE scenarios** (and it's also likely an indication there's been a failure in the
+releasing process), but if you are a contributor and have been asked to do this, here are the steps to be followed:
 
 1. On the main `ebpf-for-windows` repo, create and check out a new working branch from the "`main`" branch.
```

Comment:
A "contributor" cannot create a new branch in the main ebpf-for-windows repo, only in their own fork.  I'd recommend changing either line 137 to say maintainer instead of contributor or (my preference) change line 139 to say on one's personal fork (instead of the main repo), create a branch.

Reply:
addressed.